viadot/tasks/outlook.py

```diff
@@ -1,4 +1,4 @@
-from typing import Any, Dict, List, Tuple
+from typing import Any, Dict, List
 
```

Comment:
You can also delete `local_config` (line 8) and probably delete commented lines (from line 36). Credentials are handled in the source class.